backend/gameplay/consumers.py

```
@@ -441,7 +437,7 @@ async def connect(self):
 		connection_end = perf_counter()
 		connection_time = connection_start - connection_end
```

Ruff (0.8.2)

Local variable `connection_time` is assigned to but never used

Remove assignment to unused variable `connection_time`

(F841)